src/main/java/uk/gov/hmcts/divorce/caseworker/service/task/GenerateMiniApplication.java

```diff
@@ -31,23 +31,26 @@
     @Override
     public CaseDetails<CaseData, State> apply(final CaseDetails<CaseData, State> caseDetails) {
 
-        final Long caseId = caseDetails.getId();
-        final CaseData caseData = caseDetails.getData();
-        final LocalDate createdDate = caseDetails.getCreatedDate().toLocalDate();
-
-        log.info("Executing handler for generating mini application for case id {} ", caseId);
-
-        final Supplier<Map<String, Object>> templateContentSupplier = templateContent.apply(caseData, caseId, createdDate);
-
-        caseDataDocumentService.renderDocumentAndUpdateCaseData(
-            caseData,
-            DIVORCE_APPLICATION,
-            templateContentSupplier,
-            caseId,
-            DIVORCE_MINI_APPLICATION,
-            caseData.getApplicant1().getLanguagePreference(),
-            DIVORCE_MINI_APPLICATION_DOCUMENT_NAME + caseId
-        );
+        if (caseDetails.getData().getApplication().isSolicitorApplication()) {
+
```

Comment:
would be nice to add some logs here

Reply:
@nitinprabhu I'm not sure what else to add other than line 40?

Reply:
may be that's fine I was thinking something like log.info("Application with case id {} is solicitor application, caseId);